providers/dns/regru/regru.go

```diff
@@ -87,6 +94,18 @@ func NewDNSProviderConfig(config *Config) (*DNSProvider, error) {
 		client.HTTPClient = config.HTTPClient
 	}
 
+	if config.TlsCert != "" || config.TlsKey != "" {
```

Comment:
```suggestion
	if config.TlsCert != "" && config.TlsKey != "" {
```

Reply:
If user makes a mistake and defines only one variable he will see an error
With your suggestion he will see nothing

Reply:
I think that an explicit error is better:
```go
	
	if config.TLSCert != "" && config.TLSKey == "" || config.TLSCert == "" && config.TLSKey != "" {
		return nil, errors.New("regru: certificate or key is missing")
	}

	if config.TLSCert != "" && config.TLSKey != "" {
	// ...
```

Reply:
What about? :

```go
if config.TLSCert != "" || config.TLSKey != "" {
        if config.TLSCert == "" {
		return nil, errors.New("regru: TLS certificate is missing")
	}
        if config.TLSKey == "" {
		return nil, errors.New("regru: TLS key is missing")
	}
```

Reply:
yes, it's possible, it will increase the cyclomatic complexity but it will work.